solver/pb/ops.proto

```diff
@@ -122,7 +133,7 @@ message OpMetadata {
 	bool ignore_cache = 1;
 	// Description can be used for keeping any text fields that builder doesn't parse
 	map<string, string> description = 2;  
-	WorkerConstraint worker_constraint = 3;
+	// WorkerConstraint worker_constraint = 3;
```

Comment:
Is this deliberately left commented out?

Reply:
Yes. I'll add a better comment.